### PR TITLE
Update plotting.py

### PR DESCRIPTION
**Pull request by cjhopp**
Allow detection_multiplot to handle streams with fewer traces than templates.

### CODE REVIEW

**Comment by calum-chamberlain** [comment]
This needs an example, leaving this until post-quake has calmed a bit.

- [ ] Example to show syntax.

File: eqcorrscan/utils/plotting.py [hunk 1]
```diff
@@ -799,9 +799,7 @@ def detection_multiplot(stream, template, times, streamcolour='k',
     :param stream: Stream of data to be plotted as the background.
     :type template: obspy.core.stream.Stream
     :param template: Template to be plotted on top of the base stream.
-    :type times: list
-    :param times: list of times of detections in the order of the channels in
-                template.
+    :type times: list of detection times
```
Comment (calum-chamberlain) [inline eqcorrscan/utils/plotting.py]: This looks like it is missing a doc-string, type should just be the type.

Comment (calum-chamberlain) [inline eqcorrscan/utils/plotting.py]: Also needs an example.

**Comment by cjhopp** [comment]
See if that looks better?

**Pull request merged**